src/Common/src/Interop/Unix/System.Native/Interop.Posix_FAdvise.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        internal enum AdviseAdvice : int
```

Comment:
I'd call this FileAdvice or PosixFile even PosixFileAdvice. My PR has a MemoryAdvice to go with MAdvise.